mlrun/runtimes/base.py

```diff
@@ -216,6 +218,9 @@ def _get_db(self):
                 self._db_conn = get_run_db(self.spec.rundb, secrets=self._secrets)
         return self._db_conn
 
+    def try_auto_mount_based_on_config(self):
```

Comment:
I wanted to separate this from the `auto_mount` function, that's why I gave it a specific name - this function **does not** call `auto_mount`, so it may confuse people.

Reply:
I see, then leave a comment explaining that 🙂